OKTA-413125 Overall style edit for OIE embedded auth content - part 1 (#2571)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/ios/configlocations.md

#### Okta.plist

`Okta.plist` is a property file that stores the name and value for each configuration setting. You can find this file in the following repository directory: `okta-idx-swift/Samples/EmbeddedAuthWithSDKs/EmbeddedAuth/Okta.plist`.

#### Configuration page

> An alternative method to store the configuration settings is using the Configuration page. You can access the Configuration page from the **Configure** link on the startup page.

Reviewer: ```suggestion
An alternative method to store the configuration settings is by using the Configuration page. You can access the Configuration page from the **Configure** link on the startup page.
```

    redirectUri: "<#redirectUri#>") // Must match the redirect uri in client app settings/console
```

> For more details about different configuration options, see the Swift SDK's

Reviewer: ```suggestion
See the Swift SDK's [readme](https://github.com/okta/okta-idx-swift#readme) for more details on the different configuration options.
```

> [readme](https://github.com/okta/okta-idx-swift#readme) file.

Reviewer: ```suggestion

```

Reviewer: Removed line 37...it's all in line 36.
